.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug Report
+about: Create a report to help us improve GNU Radio
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+# 1. Have you looked in our documentation and asked the community?
+
+<!--- [Before you open an issue we suggest checking the following: -->
+<!--- - [The wiki](https://wiki.gnuradio.org/index.php/Main_Page) -->
+<!--- - [The GNU Radio Blocks documentation](https://wiki.gnuradio.org/index.php/Category:Block_Docs) -->
+<!--- - [The User manual](https://wiki.gnuradio.org/index.php/Usage_Manual) -->
+<!--- - [GNU Radio community](https://wiki.gnuradio.org/index.php/Chat) -->
+<!--- - or on [the mailing list](https://wiki.gnuradio.org/index.php/MailingLists)] -->
+
+# 2. GNU Radio version
+
+
+# 3. How did you install GNU Radio
+<!--- [Choose an option from this list -->
+<!--- - OS package manager -->
```

Comment:
Might be helpful to specify which OS and version (e.g. Ubuntu 18.04) as a secondary question

Reply:
Ah, derp, good point. Forgot that.